content/docs/start/data-versioning.md

```diff
@@ -7,8 +14,8 @@ to a different version of a 100Gb file in less than a second with a
 `git checkout`.
 
 The foundation of DVC consists of a few commands that you can run along with
-`git` to track large files and directories, or ML models. Read on, or watch our
-video guides!
+`git` to track large files, directories, or ML models. Think "Git for data".
+Read on or watch our video to learn about versioning data with DVC!
```

Comment:
Great. My only thing here is that the paragraph before the video looked nice in 2 lines but now it has a 3rd 😋 

![image](https://user-images.githubusercontent.com/1477535/96617028-b850e300-12c8-11eb-8c36-6e61bd87c62a.png)

Definitely very minor (probably nothing). We can merge this.